src/STable.cpp

```diff
@@ -1,19 +1,42 @@
 #include "STable.hpp"
 
+#include <sstream>
+
+#include "prompt.hpp"
+
 using namespace std;
 
 
 STable::STable(string filename)
 {
-    ifstream file(filename, ios::in);
-    if(file.is_open()) {
+    string puzzle_str {};
+    puzzle_str.reserve(100); // typischerweise 9*9, evtl. noch ein paar \s
```

Comment:
Try to keep the comments in English, so people outside Germany can also contribute in the future.

Reply:
Vous avez raison, bien sûr. Je vais faire attention.